src/common/components/cards/card-kebab-menu-button.scss

```diff
@@ -2,10 +2,32 @@
 // Licensed under the MIT License.
 @import '../../styles/colors.scss';
 
+.kebab-menu-callout {
+    box-shadow: 0px 6.4px 14.4px $box-shadow-132, 0px 1.2px 3.6px $box-shadow-108 !important;
```

Comment:
Can we use a `elementType.className` style instead to avoid the `!important` (friendlier to user stylesheets)?